osu.Game/Screens/Play/BeatmapMetadataDisplay.cs

```diff
@@ -49,6 +49,44 @@ public MetadataLine(string left, string right)
             }
         }
 
+        private class EpilepsyWarning : FillFlowContainer
+        {
+            public EpilepsyWarning(bool warning)
+            {
+                if (warning)
+                    this.Show();
+                else
+                    this.Hide();
```

Comment:
I don't think it makes much sense for this component to hide itself like this. I'd say it should be hidden (or just not created at all) at a higher level, in `BeatmapMetadataDisplay` itself.